src/Stratis.Bitcoin/P2P/PeerAddress.cs

```diff
@@ -12,6 +12,17 @@ namespace Stratis.Bitcoin.P2P
     [JsonObject]
     public sealed class PeerAddress
     {
+        /// <summary>
+        /// The maximum amount of times a peer can be attempted within a give time frame.
+        /// </summary>
+        internal static readonly int AttemptThreshold = 5;
+
+        /// <summary>
+        /// The amount of hours after which the peer's failed connection attempts
+        /// will be reset.
+        /// </summary>
+        internal static readonly int AttemptThresholdHours = 12;
```

Comment:
Come to think about it I would reduce the threshold, do we have a mechanism to delete a peer that was not responsive for x days (or X attempts)?

Reply:
What would reduce this to? 

No, there isn't a mechanism currently to delete peers that have been unresponsive. We did discuss this at some point but if I remember correctly, we decided against removing peers from the file?